pkg/ccl/changefeedccl/changefeed_processors.go

```diff
@@ -67,6 +69,8 @@ const EnableCloudBillingAccountingEnvVar = "COCKROACH_ENABLE_CLOUD_BILLING_ACCOU
 
 var EnableCloudBillingAccounting = envutil.EnvOrDefaultBool(EnableCloudBillingAccountingEnvVar, false)
 
+var errDatabaseTargetsChanged = changefeedbase.MarkRetryableError(errors.New("database targets changed"))
```

Comment:
Is it necessary to wrap it in a retryable error? I thought all errors were considered retryable unless explicitly marked/handled as terminal